ogr/services/github/pull_request.py

```diff
@@ -97,6 +98,20 @@ def labels(self) -> List[GithubLabel]:
     def diff_url(self) -> str:
         return f"{self._raw_pr.html_url}/files"
 
+    @property
+    def source_project(self) -> "ogr_github.GithubProject":
+        if self._source_project is None:
+            src_github_repo = self._raw_pr.head.repo
+
+            self._source_project = ogr_github.GithubProject(
+                repo=src_github_repo.name,
+                service=self._target_project.service,
+                namespace=src_github_repo.owner.login,
+                github_repo=src_github_repo,
+                read_only=self._target_project.service.read_only,
+            )
```

Comment:
Maybe I am missing something, but cannot we use the `service.get_project_from_github_repository`?

Reply:
Thanks, forgot it in the place it was made for :D